be/src/storage/lake/horizontal_compaction_task.cpp

```diff
@@ -71,16 +72,29 @@ Status HorizontalCompactionTask::execute(CancelFunc cancel_func, ThreadPool* flu
 #ifndef BE_TEST
         RETURN_IF_ERROR(tls_thread_status.mem_tracker()->check_mem_limit("Compaction"));
 #endif
+        std::vector<uint64_t> rssid_rowids;
```

Comment:
better to put `rssid_rowids` before while

Reply:
Done, and also vertical_compaction_task.cpp